integrated_channels/integrated_channel/transmitters/content_metadata.py

```diff
@@ -115,8 +114,10 @@ def _transmit_create(self, channel_metadata_item_map, content_updated_mapping):
                     f"Task failed with message [{exc.message}] and status code [{exc.status_code}]"
                 )
                 LOGGER.exception(exc)
-            else:
-                self._create_transmissions(chunk, content_updated_mapping)
+                response_status_code = exc.status_code
```

Comment:
existing clients parse the response codes >= 400 and raise a `ClientError` exception, bubbling up the status and an message inside it. this is in contrast to the learner data side where the unadulterated status and body are passed up to this method for parsing. leaving this except statement in place and extracting the status allows us to avoid having to update every existing client to stop emitting a `ClientError`.